src/vs/base/common/processes.ts

```diff
@@ -109,7 +109,7 @@ export function sanitizeProcessEnvironment(env: IProcessEnvironment, ...preserve
 	const keysToRemove = [
 		/^ELECTRON_.+$/,
 		/^VSCODE_(?!(PORTABLE|SHELL_LOGIN|ENV_REPLACE|ENV_APPEND|ENV_PREPEND)).+$/,
```

Comment:
The `.` seems like a bug 🤔 @anthonykim1 can you verify just this deleted all variables except for those explicitly mentioned in the lookahead (`?!`)?

```suggestion
		/^VSCODE_(?!(PORTABLE|SHELL_LOGIN|ENV_REPLACE|ENV_APPEND|ENV_PREPEND))$/,
```

Can you add the `_PYTHON` cases to the test?